bank-sdk/sdk/src/doc/source/integration.rst

```diff
@@ -1,6 +1,17 @@
 Integration
 ===========
 
+..
+  Audience: Android dev who integrates for the first time
+  Purpose: Describe what app configuration is needed, which preconditions have to be met, how to configure the SDK and how to run it.
+  Content type: Getting started
```

Comment:
[Clarification] Just to check it one more time `Content type: Getting started` and we are inside the Integration file, is it correct?